Templates/E-Commerce/Shopify/Shopify to HubSpot - Customers/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Shopify to HubSpot - Customers",
+    "version": "1.0.0",
+    "tags": [],
+    "additionalSystemsUsed": [
```

Comment:
Since HubSpot and Shopify don't have their own tasks, they should be included in the additional systems here. 😄